src/include/btmem.h

```diff
@@ -1111,7 +1111,7 @@ struct __wt_update {
  */
 struct __wt_update_value {
     WT_ITEM buf;
-    wt_timestamp_t start_ts;
+    wt_timestamp_t durable_ts;
```

Comment:
Does this field only used to determine global visibility?

Reply:
Yes, it is used only for global visibility check.